### ISSUE TITLE
`from torchmetrics.functional import accuracy` -- accuracy () function not working!!

### ISSUE THREAD

**Issue by rappysaha** [Wheest/pytorch-lightning-cifar#6]
I trained MobileNet-v2 using this repo. During my training, I observed following function not working in models/mobilenetv2.py file:
``` python
def evaluate(self, batch, stage=None):
    x, y = batch
    logits = self(x)
    loss = F.nll_loss(logits, y)
    preds = torch.argmax(logits, dim=1)
    acc = accuracy(preds, y)
    if stage:
        self.log(f"{stage}_loss", loss, prog_bar=True)
        self.log(f"{stage}_acc", acc, prog_bar=True)
```
I need to change it as following:

``` python
def evaluate(self, batch, stage=None):
    x, y = batch
    logits = self(x)
    loss = F.nll_loss(logits, y)
    preds = torch.argmax(logits, dim=1)
    acc = accuracy(preds, y, task="multiclass", num_classes=10)
    if stage:
        self.log(f"{stage}_loss", loss, prog_bar=True)
        self.log(f"{stage}_acc", acc, prog_bar=True)
```

I am using:
- torch                    2.2.1
- lightning-bolts          0.7.0
- pytorch-lightning        1.9.5
